python-package/README.rst

```diff
@@ -163,6 +163,15 @@ By default, installation in environment with 32-bit Python is prohibited. Howeve
 
 It is **strongly not recommended** to use this version of LightGBM!
 
+Build with time costs output
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: sh
+
+    pip install lightgbm --install-option=--time-costs
+
+Users who want to perform benchmarking can make LightGBM output time costs for different internal routines thanks to this install parameter
```

Comment:
```suggestion
Use this option to make LightGBM output time costs for different internal routines, to investigate and benchmark its performance.
```

Will you please switch to this wording which is a bit more direct (e.g. "Use this..." instead of "Users who...")? It's a minor personal preference, but I prefer language like that for documentation like this.

Reply:
Alright! FYI, I've mostly copy/pasted it from the [installation guide](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html) to remain coherent. In case you would like to change it there too.